### ISSUE TITLE
Ignore email addresses during spellcheck

### ISSUE THREAD

**Issue by rajsite** [streetsidesoftware/vscode-spell-checker#257]
When used with markdown files that contain emails the spellchecker flags email addresses. In my use case these were html comments containing markdown with emails embedded in markdown documents.

ie:
```markdown
## Title
<!--
For more info contact: notrealwordz@domain.com
-->
```

And in that example notrealwordz will be reported as misspelled. It does not seem to matter if the emails are encoded for automatic links, ie the following still shows a spelling error: `<notrealwordz@domain.com>`

**Comment by Jason3S** [comment]
You can turn on ignore emails:

In your user or workplace settings, you can tell the spell checker to ignore emails for `markdown`:
```
"cSpell.languageSettings": [
    {
        "languageId": "markdown",
        "ignoreRegExpList": ["Email"]
    }
]
```

To always ignore emails, this will work:

```
"cSpell.ignoreRegExpList": ["Email"]
```

**Comment by rajsite** [comment]
Thanks for the information, I'll try it out if I run into that behavior again.
